cedar-policy-validator/src/types.rs

```diff
@@ -401,7 +480,8 @@ impl Type {
                 primitive_type: Primitive::Bool,
             } => Type::json_type("Boolean"),
             Type::Primitive {
-                primitive_type: Primitive::Long,
+                // FIXME: Preserve bounds? What uses this?
```

Comment:
I added the bounds here, but it looks like for the error messages we currently have, printing the bounds may just be distracting and not helpful.  For example, for an erroneous expression `true > 1`, instead of:
```
Unexpected type. Expected one of [{"type":"Long"}] but saw {"type":"True"}
```
we now get:
```
Unexpected type. Expected one of [{"type":"Long","min":-9223372036854775808,"max":9223372036854775807,"can_be_any":true}] but saw {"type":"True"}
```
(I see two other error messages that include types, but I'm not aware of a way to trigger either of them with a `Long` type.)  So I'm thinking the right thing to do for the user is leave out the bounds, even though it seems inconsistent design-wise.  If in the future, this function is used in a context in which the bounds are helpful, we would reevaluate.